public/css/global.css

```diff
@@ -1,4 +1,12 @@
-html, body {
-    margin: 0;
-    padding: 0;
+body {
+  margin: 0;
+  padding: 0;
+  background-color: lightblue;
+}
+
+.title {
```

Comment:
"Title" is a very general name, so if in the future we would make some other objects with the `className` "title" it will affect the CSS. So I think that making a variable for the CSS would be the best. Also, try using `Typography` from MaterialUI (is way easier 😉).